README.md

```diff
@@ -49,6 +49,10 @@ Show a course
 moodle.courses.show(course_id)
 ```
 
+```
+moodle.courses.show(course_ids) #Example: moodle.courses.show([1030, 679....])
```

Comment:
We should update this example, because if you directly pass an array, that will come out as parameter 1.
```suggestion
moodle.courses.show(course_ids) # Example: moodle.courses.show(1234, 234, ...)
```